velox/common/caching/SsdFile.cpp

```diff
@@ -169,11 +169,13 @@ SsdFile::SsdFile(const Config& config)
   writeFile_ = fs_->openFileForWrite(fileName_, fileOptions);
```

Comment:
I'd put this in openFileForWrite when we construct a file to call this? Then we don't need a separate API to support in velox fs interface. We could add a new open file option? Thanks!